src/main/java/core/mate/academy/model/BulldozerProducer.java

```diff
@@ -0,0 +1,14 @@
+package core.mate.academy.model;
+
+import core.mate.academy.service.MachineProducer;
+import java.util.ArrayList;
+import java.util.List;
+
+public class BulldozerProducer implements MachineProducer<Bulldozer> {
```

Comment:
It's better to create impl package in service package and replace BulldozerProducer and other producers there